### PR TITLE
Create Request AED user

### PR DESCRIPTION
**Pull request by BeluMymoid**
New template for: Add, edit or delete user process.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Request AED user [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Issue Title: [Addition / Edit / Delete] User.
+
+### Affected User(s): 
+- [Full name(s) of affected user(s)]
+- [Full name(s) of affected user(s)]
+- [Full name(s) of affected user(s)
+
+### Department: Department the user belongs to or would belong to
+
+###  Change Type: [Addition / Modification / Removal] of [user/permissions in application/permissions in PCI-DSS environment]
```
Comment (antoniojavier-mymoid) [inline .github/ISSUE_TEMPLATE/Request AED user:10]: I would indicate that "PCI-DSS environment" is optional

File: .github/ISSUE_TEMPLATE/Request AED user [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Issue Title: [Addition / Edit / Delete] User.
+
+### Affected User(s):
```
Comment (antoniojavier-mymoid) [inline .github/ISSUE_TEMPLATE/Request AED user:3]: I prefer one issue per user, but it could be good test massive updates. So it is nice too

Comment (BeluMymoid) [reply .github/ISSUE_TEMPLATE/Request AED user:3]: As you say, I think it is better to have one per user, to have better control (and follow-up). We change it.

File: .github/ISSUE_TEMPLATE/Request AED user (hunk 1 above)
Comment (BeluMymoid) [reply .github/ISSUE_TEMPLATE/Request AED user:10]: oks

File: .github/ISSUE_TEMPLATE/Request AED user (hunk 2 above)
Comment (BeluMymoid) [reply .github/ISSUE_TEMPLATE/Request AED user:3]: As you say, I think it is better to have one per user, to have better control (and follow-up). We change it.

**Review by antoniojavier-mymoid — APPROVED** [review]
After review, it is approved

**Pull request merged**